src/Elsa.Dashboard/Pages/AccessDenied.cshtml.cs

```diff
@@ -0,0 +1,12 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Elsa.Dashboard.Pages
```

Comment:
Only just spotted this, and didn't realise it existed before - and it looks like the other pagemodels have this code-behind file too.  May be worth moving the PageModels to these, rather than their own explicit classes?